_docs/tasks/security/mutual-tls.md

```diff
@@ -82,21 +75,39 @@ There are several steps:
 
    Make sure the pod is "Running".
 
-1. ssh into the envoy container 
+1. ssh into the envoy container
    ```bash
    kubectl exec -it productpage-v1-4184313719-5mxjc -c istio-proxy /bin/bash
    ```
 
 1. make sure the key/cert is in /etc/certs/ directory
    ```bash
-   ls /etc/certs/ 
+   ls /etc/certs/
    ```
    ```bash
    cert-chain.pem   key.pem   root-cert.pem
-   ``` 
-   
-   Note that cert-chain.pem is envoy's cert that needs to present to the other side. key.pem is envoy's private key paired with cert-chain.pem. root-cert.pem is the root cert to verify the other side's cert. Currently we only have one CA, so all envoys have the same root-cert.pem.  
-   
+   ```
+
+   Note that cert-chain.pem is envoy's cert that needs to present to the other side. key.pem is envoy's private key paired with cert-chain.pem. root-cert.pem is the root cert to verify the other side's cert. Currently we only have one CA, so all envoys have the same root-cert.pem.
+
+1. make sure 'curl' is installed by
+   ```bash
+   curl
```

Comment:
I guess you could put
```
curl localhost:15000
```
or some such (the admin port for the proxy)

Reply:
I am afraid that would be misleading, people may wonder what is this 15000? And sometimes if 15000 is down for some reason, it will mislead people.